<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>611</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="shownPic">
        <property name="minimumSize">
         <size>
          <width>371</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>99999</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>420</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>420</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="shownFreqPic">
        <property name="minimumSize">
         <size>
          <width>371</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>99999</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="preview_filter_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="filter_1_btn">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Gaussion Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="preview_filter_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="filter_2_btn">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Median Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="preview_filter_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="filter_3_btn">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Low Pass Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="preview_filter_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="filter_4_btn">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>High Pass Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="preview_filter_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="filter_5_btn">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Histogram Eqaulization</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="showHistBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Show Hist.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>48</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="imageNamesComboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_20">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>48</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item alignment="Qt::AlignLeft">
             <widget class="QRadioButton" name="spatialFiltersRadioBtn">
              <property name="maximumSize">
               <size>
                <width>153</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>Spatial Domain</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QRadioButton" name="freqFiltersRadioBtn">
              <property name="maximumSize">
               <size>
                <width>153</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>Frequency Domain</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_File_as_greyscale"/>
   </widget>
   <widget class="QMenu" name="menutheme">
    <property name="title">
     <string>theme</string>
    </property>
    <addaction name="actiondark"/>
    <addaction name="actionlight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menutheme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_File_as_greyscale">
   <property name="text">
    <string>Open File as greyscale</string>
   </property>
  </action>
  <action name="actiondark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>dark</string>
   </property>
  </action>
  <action name="actionlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>light</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
